util/design/get-lfsr-coeffs.py

```diff
@@ -119,14 +119,15 @@ def main():
         if args.fib:
             lfsrType = 'FIB_XNOR'
 
-            wget.download(FIB_URL)
-            cmd = ['pdftotext %s.pdf' % PDF_NAME, '> %s.txt' % PDF_NAME]
+            if not os.path.isfile(args.fib):
+                print("File %s does not exist" % args.fib)
+                sys.exit(1)
+
+            cmd = ['pdftotext %s %s/xapp052.txt' %
+                   (os.path.abspath(args.fib), os.getcwd())]
             subprocess.call(cmd, shell=True)
```

Comment:
I'm not sure you need `shell=True` any more, since you've dropped the redirection?

Reply:
Hm... it does not seem to find the `pdftotext` utility if I remove `shell=True`.

Reply:
Oh, silly me! I forgot that we're also using the shell for resolving (in this case) `pdftotext` to `/usr/bin/pdftotext`. Good point!